openpype/plugins/publish/extract_review_slate.py

```diff
@@ -122,10 +122,14 @@ def process(self, instance):
                     if stream["channel_layout"]:
                         audio_channel_layout = str(
                             stream.get("channel_layout"))
+                    if stream["codec_name"]:
+                        audio_codec = str(
+                            stream.get("codec_name"))
```

Comment:
Value is used for check if there is a codec. But if `None` is converted to string it will be `"None"` which will pass the condition even if there is not information about it.
```suggestion
                        audio_codec = stream.get("codec_name")
```
The same should be changed for `channel_layout` `sample_rate` and `channels`.

Reply:
will do